README.md

```diff
@@ -1,6 +1,6 @@
 # Extension function
 
-Your task is to add a new function `getSizeMessage() : String` for `ArrayList` class. 
+In this task, you need to add a new function, `getSizeMessage(): String,` to the ArrayList class. 
```

Comment:
In this task, you need to add a new function, `getSizeMessage(): String` to the `ArrayList` class.